org.eclipse.xtend.ide/src/org/eclipse/xtend/ide/buildpath/Junit5LibClasspathAdder.java

```diff
@@ -37,5 +34,21 @@ protected IClasspathEntry createContainerClasspathEntry() {
 	protected String[] getBundleIds() {
 		return BUNDLE_IDS;
 	}
+	
+	@Override
+	protected String[] getImportedPackages() {
+		return new String[] {
+			"org.junit.jupiter.api;version=\"[5.1.0,6.0.0)\"",
```

Comment:
wouldn't it be better to have reusable constants for version ranges? See also above...

Reply:
am not sure if this would make the code any better readable / maintainable
if needed it can be done in follow up

Reply:
OK, no problem